### DISCUSSION TITLE
Integrate Govee temperature/humidity sensor?

### DISCUSSION THREAD

**Discussion by mstormo** [mKeRix/room-assistant#449] [Ideas]
Any chance someone would be able to integrate the Govee sensor into Room Assistant?

Although the python code for the Home Assistant component looks fairly simple, I have not touched a single TypeScript line of code in my life. :)

https://github.com/Home-Is-Where-You-Hang-Your-Hack/sensor.goveetemp_bt_hci

**Comment by mKeRix** [comment]
Unfortunately I don't own one of these and feel like implementing this without a way of testing it myself would be too time consuming... but I'm happy to accept a PR for it if someone else feels like working on this! The custom component you linked isn't super complicated indeed.
